tests/unit/database_tests.py

```diff
@@ -640,6 +640,7 @@ def setUp(self):
         """
         super(CloudantDatabaseTests, self).setUp()
         self.db_set_up()
+        self.create_search_index()
```

Comment:
Is it worth having this in the `setUp` for 6 tests?  Maybe move it to the tests themselves?